src/response.cc

```diff
@@ -1,32 +1,40 @@
 #include "response.h"
 
-std::string status_code_to_string(int status_code);
+void Response::SetStatus(const ResponseCode response_code) {
+  switch (response_code) {
+    case OK: {
+      status_ = "HTTP/1.1 200 OK";
+      break;
+    }
+    case NOT_FOUND: {
+      status_ = "HTTP/1.1 404 Not Found";
+      break;
+    }
+    case INTERNAL_SERVER_ERROR:
+    default: {
+      status_ = "HTTP/1.1 500 Internal Server Error";
+      break;
+    }
+  }
+}
+  
+void Response::AddHeader(const std::string& header_name, const std::string& header_value) {
```

Comment:
to be consistent with the other method names, can you put the second parameter on the next line?